sirepo/package_data/static/js/cloudmc.js

```diff
@@ -805,11 +834,12 @@ SIREPO.app.directive('geometry2d', function(appState, cloudmcService, frameCache
                         Math.abs(ranges[m][1] - ranges[m][0]) / Math.abs(ranges[l][1] - ranges[l][0])
                     )
                 );
+                tallyService.updateThresholds();
                 const r =  {
                     aspectRatio: ar,
-                    global_max: tallyService.maxField,
-                    global_min: tallyService.getMinWithThreshold(),
-                    threshold: appState.models.openmcAnimation.threshold,
+                    global_min: appState.models.openmcAnimation.thresholds.global[0],
+                    global_max: appState.models.openmcAnimation.thresholds.global[1],
+                    threshold: appState.models.openmcAnimation.thresholds.val,
```

Comment:
I think it the plot is more meaningful if you match the colorscale to the selected threshold, rather than the global min max:

global colorscale range:
![x](https://github.com/radiasoft/sirepo/assets/6516910/5bb7c1e0-9525-45a0-98ee-85873c2a35ff)

threshold colorscale range:
![x2](https://github.com/radiasoft/sirepo/assets/6516910/9dd82fea-1463-4539-b476-3f66849bdd55)

Reply:
[27f93d3](https://github.com/radiasoft/sirepo/pull/6781/commits/27f93d3d43b6ba6fea5b3898b31a02b219c6f53e)